pkg/providers/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -51,7 +51,7 @@ func (e EKS) Script() (string, error) {
 	return base64.StdEncoding.EncodeToString([]byte(strings.ReplaceAll(userData, "\r", ""))), nil
 }
 
-//nolint:gocyclo
+// nolint:gocyclo
```

Comment:
I thought this had to be what it was before?

Reply:
I can change it back, looks like formatting may have changed this for some reason 🤔